aten/src/ATen/templates/Operators.h

```diff
@@ -2,6 +2,13 @@
 
 // ${generated_comment}
 
+#ifdef TORCH_ASSERT_NO_OPERATORS
+#error This change adds a dependency on native_functions.yaml,             \
+  meaning the file will need to be re-compiled every time an operator      \
+  is changed or added. Consider if your change would be better placed in   \
+  another file, or if a more specific header might achieve the same goal.
```

Comment:
I'm not sure that this error message will be helpful for people that don't have context on this PR.
Maybe we want to extend a bit the comment next to TensorBase definition and link it from here.